extensions/podman/src/podman-install.ts

```diff
@@ -410,18 +414,24 @@ class WinVersionCheck extends BaseCheck {
       if (Number.parseInt(winBuild) >= this.MIN_BUILD) {
         return { successful: true };
       } else {
-        return this.createFailureResult(
-          'To be able to run WSL2 you need Windows 10 Build 18362 or later.',
-          'WSL2 Install Manual',
-          'https://docs.microsoft.com/en-us/windows/wsl/install-manual#step-2---check-requirements-for-running-wsl-2',
-        );
+        return this.createFailureResult({
+          description: 'To be able to run WSL2 you need Windows 10 Build 18362 or later.',
+          docLinksDescription: 'Learn about WSL requirements:',
+          docLinks: {
+            url: 'https://docs.microsoft.com/en-us/windows/wsl/install-manual#step-2---check-requirements-for-running-wsl-2',
+            title: 'WSL2 Install Manual',
```

Comment:
I know it's not related to this PR, just wondering if `WSL2 Install Manual` is correct or it should be `WSL2 Manual Installation Steps`

Reply:
Renamed. I don't have a strong opinion on this. Both works fine on my end but if you have a preference for the second, it's ok for me.